src/codegen/generators/expression_generator.rs

```diff
@@ -475,6 +475,7 @@ impl<'ink, 'b> ExpressionCodeGenerator<'ink, 'b> {
                     Err(Diagnostic::codegen_error("Negated expression must be numeric", expression))
                 }
             }
+            Operator::Plus => self.generate_expression(expression),
```

Comment:
Not a fan of having this in the codegen because it's a noop but there's no other way because the function expects some return value right?